backend/apps/ai/management/commands/ai_create_project_chunks.py

```
@@ -0,0 +1,225 @@
+            prose_parts.append(f"Summary: {project.summary}")
+
+        if project.level:
+            metadata_parts.append(f"Project Level: {project.level}")
+
+        if project.type:
+            metadata_parts.append(f"Project Type: {project.type}")
+
+        if hasattr(project, "owasp_repository") and project.owasp_repository:
+            repo = project.owasp_repository
+            if repo.description :
+              prose_parts.append(f"Repository Description: {repo.description}")
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)